docs/debugging-install.md

```diff
@@ -0,0 +1,59 @@
+# Debugging Installation Issues
+
+## General
+
+#### `invalid configuration: no configuration has been provided`
+This typically means that no `kubeconfig` file can be found for the Ark client to use. Ark looks for a kubeconfig in the 
+following locations:
+* the path specified by the `--kubeconfig` flag, if any
+* the path specified by the `$KUBECONFIG` environment variable, if any
+* `~/.kube/config`
+
+#### Backups or restores stuck in `New` phase
+This implies that the Ark server is not up and running. Check the pod description and logs for errors:
```

Comment:
s/implies/means ? "implies" is a little vague, suggests that other things might cause the issue. IF this is indeed the case (other things), we might want to mention them. If it's not, then "means" is the right word.

Reply:
It's *possible* that there could be other causes for this, but what's described here is definitely the most common and the only one I care to capture. Should I just use "means"? "almost always means"?

Reply:
"means" is good. We can change it if an edge case shows up and we should doc it ;)